helm/mediator/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.deployment_name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.deployment_name }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.deployment_name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.deployment_name }}
+    spec:
+      containers:
+      - name: {{ .Values.container.name }}
+        image: " {{ .Values.image.name }}:{{ .Values.image.tag }} "
+        imagePullPolicy: Always
```

Comment:
I think we should add the ability to control resource usage. Able to add that to the deployment?

```yaml
          resources:
            {{- toYaml .Values.resources | nindent 12 }}
```

Then add `values_dev.yml` with the following:

```yaml
resources:
  requests:
    memory: 256Mi
    cpu: 20m
  limits:
    memory: 256Mi
    cpu: 100m
```